docs(readme): align with website and add branding (#7) (#39)
rekwai/rekwai: docs/queries.md

### Step 1: Upload Your Questionnaire

1. Go to the Queries tab in your product
2. Click the "Upload" dropdown and select "Upload Query"

> 3. Choose a questionnaire document (PDF, Word `.docx`, Excel, Markdown, or plain text)

Reviewer: I would not put the file types here, because it'll be hard to keep the documentation up to date. If the `frontend/components/query/upload-query-dialog.tsx` functionality mentions this to the user, I think that's enough.

```suggestion
3. Choose a questionnaire document that is supported by Rekwai
```
